nodebuilder/share/cmd/share.go

```diff
@@ -156,3 +158,36 @@ var getEDS = &cobra.Command{
 		return cmdnode.PrintOutput(shares, err, nil)
 	},
 }
+
+var getRange = &cobra.Command{
+	Use:   "get-range [height] [start] [end(exclusive)]",
+	Short: "Gets a range of shares from the given height within the given ODS indexes",
+	Args:  cobra.ExactArgs(3),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		client, err := cmdnode.ParseClientFromCtx(cmd.Context())
+		if err != nil {
+			return err
+		}
+		defer client.Close()
+
+		height, err := strconv.ParseUint(args[0], 10, 64)
+		if err != nil {
+			return err
+		}
+		start, err := strconv.ParseUint(args[1], 10, 64)
+		if err != nil {
+			return err
+		}
+		end, err := strconv.ParseUint(args[2], 10, 64)
+		if err != nil {
+			return err
+		}
+
+		if start >= end {
+			return errors.New("start must be greater than end")
```

Comment:
error message is confusing. should be `errors.New("start index must be less than end index")`

Reply:
updated